### PR TITLE
Create an alternative 08B instead of overwriting the existing one

### PR DESCRIPTION
**Pull request by freemansoft**
Create an alternative to 08_device_subclassing that moves the remote tasks into their own file that could be used in either example. Tested on windows both programs with two RP2040 boards.

Changed the LED for the RP2040 to be board independent. Pin25 is the LED pin only on the non-wifi board. This assumes you are using a version of MicroPython 1.19 from Jan 2023.

### CODE REVIEW

**Comment by BrianPugh** [comment]
So I think this is better, but still needs a few things before merging:

1. I need to add a `contributing.md` guide to this repo, but basically we need to run linting before this can be merged (See enum 2 under Environment Setup [here](https://github.com/BrianPugh/python-template/blob/main/.github/contributing.md)).

2. I think the README additions are good; I was wondering if you could copy all but the last paragraph over to the origianl example 8? For 8b, can you move the last paragraph up to the top of the README? I would adjust it to be like:

```
This is an alternative version of Example 08 that explicitly separates
code that runs on the host (``main.py``, ``main_multiple_devices``)
from the class that contains the code (``MyDevice`` in ``mydevice.py``)
that will be pushed to the microcontroller.
```

**Review by BrianPugh — APPROVED** [review]

**Comment by BrianPugh** [comment]
thanks! I'll merge after CI is done running.

**Comment by freemansoft** [comment]
I tried running a linter against the code tree but did not find the trailing spaces caught in the automated build. Ran the linter in VS Code.  Couldn't get poetry running correctly on my PC.  My try my mac later

Saw I missed the 08B instead O8 at the top of README.rst

Wasn't sure about the trailing spaces in the README.rst if they were important

**Comment by freemansoft** [comment]
I see an isort failure but don't understand what that means.

**Comment by BrianPugh** [comment]
all good, I pushed the linting fixes. `isort` is a program that just makes sure imports are sorted alphabetically to try and make imports more organized.

**Comment by freemansoft** [comment]
I ran flake8 against that and it didn't see the import issue :-(

**Comment by BrianPugh** [comment]
flake8 is just one of the many checkers we run. If you install `pre-commit`, then run `pre-commit install` in this repo, it will automatically run all of our checkers configured in `.pre-commit-config.yaml` on staged files before every commit. If anything fails, it'll not allow the commit. Some of the checkers, like `black` and `isort` will automatically fix the files. Some others won't (since they may require actual code/logic changes). To run pre-commit on all files in a repo without commiting, you can run `pre-commit run --all`.

**Comment by freemansoft** [comment]
I couldn't get pieces to work correctly when I tried earlier.

Of course now when I run `pre-commit run --all` it ran all the checkers and resorted the imports.  ¯\_(ツ)_/¯

**Comment by BrianPugh** [comment]
merged; thank you!

**Pull request merged**
